JsonListener.h: No such file or directory

In file included from src/SegwitClient.cpp:3:0:
src/SegwitClient.h:4:26: fatal error: JsonListener.h: No such file or directory
#include <JsonListener.h>
^
compilation terminated.

--- a/platformio.ini
+++ b/platformio.ini
@@ -12,3 +12,4 @@
 platform = espressif8266
 board = nodemcuv2
 framework = arduino
+lib_deps = JsonStreamingParser, ESP8266_SSD1306, WifiManager

